test/smoke/src/areas/terminal/terminal-shellIntegration.test.ts

```diff
@@ -25,7 +25,9 @@ export function setup() {
 					await terminal.assertShellIntegrationActivated();
 				});
 			});
-			if (!process.platform.startsWith('win')) {
+			if (process.platform === 'win32') {
```

Comment:
Doing it inline with a ternary statement is how we typically do this, that way the 'Decorations' only appears once:

Reply:
still not sure I did this correctly 🤔 . I see `Decorations` appearing more than once in any arrangement and don't see examples of this elsewhere

Reply:
oh figured it out